### PR TITLE
docs: develop-guide add multilingual translation process

### PR DESCRIPTION
**Pull request by ziqiangai**
Submit a pull request for this project.

# Why? 

Add guidance on multilingual translations for contributors

# What?

Contributors get clearer guidance.
# How?

### CODE REVIEW

File: docs/contribute/developer-guide.md [hunk 1]
```diff
@@ -131,6 +131,17 @@ Just open APITable's root directory with IDE.
 
 
 
+## How to contribute translations?
+
+We have two ways to improve the translation of APITable:
+
+1. Modify the source code and create a PR
```
Comment (mr-kelly) [inline docs/contribute/developer-guide.md]: You can modify the markdown files in source code and create a PR directly.

File: docs/contribute/developer-guide.md [hunk 2]
```diff
@@ -131,6 +131,17 @@ Just open APITable's root directory with IDE.
...
+2. Find the `strings` to be modified in [Crowdin](https://crowdin.com/project/apitablecode) and modify them online
```
Comment (mr-kelly) [inline docs/contribute/developer-guide.md]: Join our [Crowdin](https://crowdin.com/project/apitablecode) to find the `strings` to modify

File: docs/contribute/developer-guide.md (hunk 1 above)
Comment (ziqiangai) [reply docs/contribute/developer-guide.md]: But there may still be json files that need to be translated in the future.

**Review by yort-feng — APPROVED** [review]

File: docs/static/collaboration_of_multilingual_translation.png
Comment (mr-kelly) [inline docs/static/collaboration_of_multilingual_translation.png:1]: Simplified the "Crowdin Online Editor" column

**Review by mr-kelly — APPROVED** [review]

**Pull request merged**